### ISSUE TITLE
Configuration wizard hangs 

### ISSUE THREAD

**Issue by shrx** [pimalaya/himalaya#517]
```
~/.local/bin/himalaya.exe --version
himalaya v1.0.0 +smtp +sendmail +maildir +wizard +pgp-commands +imap
build: windows gnu x86_64
git: nix-flake-20241209110415, rev ce0b2dd8d378c834bf22600d4ea508baad43dab0
```

```
 ~/.local/bin/himalaya.exe
Cannot find configuration at C:\Users\sharix\AppData\Roaming\himalaya\config.toml.
? Would you like to create one with the wizard? (Y/n) Y
# here it just hangs
```

Tested in local MobaXterm terminal with zsh.

**Comment by soywod** [comment]
Did you try different combinations, like straight `Enter`, `y` then `Enter`, `Y` then `Enter`?

**Comment by shrx** [comment]
Yes, same result in all cases. Additionally, it seems that the argument is not passed to the program:

```
~/.local/bin/himalaya.exe
Cannot find configuration at C:\Users\sharix\AppData\Roaming\himalaya\config.toml.
? Would you like to create one with the wizard? (Y/n) y
# here I press ctrl+c
y
zsh: y: command not found.
```

**Comment by soywod** [comment]
> Yes, same result in all cases. Additionally, it seems that the
> argument is not passed to the program

Himalaya relies on a crate called inquire to prompt stuff to terminal,
could you try to run one of their example?

  <https://github.com/mikaelmello/inquire/tree/main/inquire/examples>

Something like confirm.rs should be enough:

  cargo run --example confirm

**Comment by soywod** [comment]
It may also be an issue with your $TERM environment variable. What is
its actual value for you?

**Comment by shrx** [comment]
`echo $TERM` returns `xterm`. Will try to test the inquire example now.

**Comment by soywod** [comment]
Definitely looks like an issue with `inquire` not supporting your terminal. Any chance for you to try with another one? If the example fails, let me know so I can open an issue on their repository.
